src/lib/eventStack/eventStack.tsx

```diff
@@ -3,13 +3,13 @@ import EventTarget from './EventTarget'
 import normalizeTarget from './normalizeTarget'
 
 class EventStack {
-  _targets = new Map()
+  private readonly _targets = new Map()
 
   // ------------------------------------
   // Target utils
   // ------------------------------------
 
-  _find = (target, autoCreate = true) => {
+  private _find = (target, autoCreate = true) => {
```

Comment:
can we add also typings for methods' parameters, in regards to refactoring of event stack?

Reply:
btw, you might consider it as separate PR

Reply:
would rather focus on functionality in this PR - those haven't existed before, would like to devote PR to introduce them